crates/aptos/src/common/types.rs

```diff
@@ -1181,7 +1181,7 @@ pub struct MovePackageDir {
     pub language_version: Option<LanguageVersion>,
 
     /// Select bytecode, language version, and compiler to support Move 2:
-    /// Same as `--bytecode_version=7 --language_version=2.0 --compiler_version=2.0`
+    /// Same as `--bytecode_version=7 --language_version=2.1 --compiler_version=2.0`
```

Comment:
In the attributes for the field `language_version` above, we have `default_value_if("move_2", "true", "2.0")`: should we change this to `default_value_if("move_2", "true", "2.1")`?